rtd_poetry.py

```diff
@@ -4,10 +4,10 @@
 from pathlib import Path
 
 from poetry.masonry.api import *
-from poetry.poetry import Poetry
+from poetry.factory import Poetry
```

Comment:
This in fact should read `from poetry.factory import Factory`.

Reply:
Thank you.  Fixed.  My bad.